server/include/server/Server.hpp

```diff
@@ -88,7 +209,13 @@ class Server {
     boost::asio::steady_timer tick_timer_;
     bool running_;
 
-    static constexpr int TICK_RATE_MS = 16;  // ~60 FPS
+    // Active client connections mapped by PlayerId
+    std::unordered_map<uint8_t, ClientConnection> clients_;
+    // Next available PlayerId
+    uint8_t next_player_id_;
+
+    static constexpr int TICK_RATE_MS = 16;    // ~60 FPS
+    static constexpr uint8_t MAX_CLIENTS = 4;  // R-Type supports 4 players
```

Comment:
The PlayerID variable can go from 0 to 255

```suggestion
    static constexpr uint8_t MAX_CLIENTS = 255;  // R-Type supports 255 players
```